storybook/stories/API/cartesian/ReferenceArea.stories.tsx

```diff
@@ -45,7 +47,7 @@ const StyleProps: Args = {
     },
   },
   isFront: {
-    description: `If set true, the line will be rendered in front of bars in BarChart, etc.`,
+    description: `If set true, the area will be rendered in front of bars in BarChart, etc.`,
```

Comment:
If we use a more generic description, i.e. 
```suggestion
    description: `If set true, the component will be rendered in front of bars in BarChart, etc.`,
```

Then we could reuse this across all places where it is used.

Reply:
Thanks for the feedback! That's a good idea, I'll extract the arg type.